datapacks/Nomad Train/data/nomad_train/functions/the_nether.mcfunction

```diff
@@ -0,0 +1,8 @@
+#Removing forceload from previous chunk, setting marker's Pos value to Pos from nt:worldborder_center_marker storage and forceloading that chunk
```

Comment:
comment - please add info why forceload is needed

Reply:
done